File: gluecodium/src/test/resources/smoke/locales/output/android/com/example/smoke/LocaleDefaults.java
/*
 *
 */
package com.example.smoke;
import android.support.annotation.NonNull;
import java.util.Locale;
public final class LocaleDefaults {
    @NonNull
    public Locale english;
    @NonNull
    public Locale latAmSpanish;
    @NonNull
    public Locale romanshSursilvan;
    @NonNull
    public Locale serbianCyrillic;
    @NonNull
    public Locale traditionalChineseTaiwan;
    @NonNull
    public Locale zuerichGerman;
    public LocaleDefaults() {
        this.english = Locale.forLanguageTag("en");
        this.latAmSpanish = Locale.forLanguageTag("es-419");
        this.romanshSursilvan = Locale.forLanguageTag("rm-sursilv");
        this.serbianCyrillic = Locale.forLanguageTag("sr-Cyrl");
        this.traditionalChineseTaiwan = Locale.forLanguageTag("nan-Hant-TW");
        this.zuerichGerman = Locale.forLanguageTag("gsw-u-sd-chzh");
    }
}

File: gluecodium/src/test/resources/smoke/locales/output/android-kotlin/com/example/smoke/LocaleDefaults.kt
/*

 *
 */

@file:JvmName("LocaleDefaults")

package com.example.smoke

import java.util.Locale

class LocaleDefaults {
    @JvmField var english: Locale
    @JvmField var latAmSpanish: Locale
    @JvmField var romanshSursilvan: Locale
    @JvmField var serbianCyrillic: Locale
    @JvmField var traditionalChineseTaiwan: Locale
    @JvmField var zuerichGerman: Locale



    constructor() {
        this.english = Locale.forLanguageTag("en")
        this.latAmSpanish = Locale.forLanguageTag("es-419")
        this.romanshSursilvan = Locale.forLanguageTag("rm-sursilv")
        this.serbianCyrillic = Locale.forLanguageTag("sr-Cyrl")
        this.traditionalChineseTaiwan = Locale.forLanguageTag("nan-Hant-TW")
        this.zuerichGerman = Locale.forLanguageTag("gsw-u-sd-chzh")
    }





}

